internal/operators/cnv/cnv_operator_test.go

```diff
@@ -352,21 +353,47 @@ var _ = Describe("CNV operator", func() {
 	})
 
 	Context("cluster requirements", func() {
-		It("only x86_64 is supported for CNV operator", func() {
```

Comment:
It seems that all the new tests added to this file are already covered by `features_olm_operator_test`. Since the tested logic resides in `features_olm_operator` I think most of the added tests here should be removed ad keep just one "supported" and one "unsupported" scenario to be tested here.

Reply:
The test here are for the CNV opertor functions - The test in `featuresupport` are for the calls made to features support which calls `operators.manager` which than includes the operator calls.

so we need both casue the the feature support goes throw a few more checks in the way.

as an example here we tests `cnv.ValidateCluster` but the feature support goes throw `manager.ValidateCluster` which does some general checks and than calls `cnv.ValidateCluster`
https://github.com/openshift/assisted-service/blob/master/internal/operators/manager.go#L289